.github/ISSUE_TEMPLATE/helm_bug_report.yaml

```diff
@@ -0,0 +1,56 @@
+---
+name: Ngrok Ingress Controller Helm Chart Bug Report
+labels: ["bug", "area/helm-chart", "needs-triage"]
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this bug report!
+
+      Note, you do not need to create an issue if you have a change ready to submit.
+      You can open a [pull request](https://github.com/ngrok/ngrok-ingress-controller/pulls) immediately instead.
+- type: input
+  attributes:
+    label: Kubernetes Version
+    description: Which Kubernetes Version do you use?
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Helm Chart Version
+    description: Which version of the Helm chart do you use?
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Helm Chart configuration
+    description: Additional description of your Helm Chart configuration.
+    placeholder: >
+      Enter any relevant details of your Helm Chart configuration. Maybe you can
+      paste your `values.yaml` or important parts of it here? Make sure to surround the code
+      you paste with ``` ```.
+- type: textarea
+  attributes:
+    label: What happened
+    description: Describe what happened.
+    placeholder: >
+      Please provide the context in which the problem occurred and explain what happened
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What you think should happen instead
+    description: What do you think went wrong?
+    placeholder: >
+      Please explain why you think the behavior you observed is incorrect. It is extremely helpful
+      if you copy&paste the fragment of logs showing the exact error messages or wrong behavior.
+- type: textarea
+  attributes:
+    label: How to reproduce
+    description: What should we do to reproduce the problem?
+    placeholder: >
+      Please provide a reproducible step-by-step case of how to reproduce the problem as minimally and
+      precisely as possible. It will greatly help us in figuring out the root cause of the problem and fixing it.
+- type: markdown
+  attributes:
+    value: "Thank you for help in improving the Ngrok Ingress Controller Helm Chart!"
```

Comment:
nit: `Ngrok` is always lower case is my understanding. You can see on the main site its always just `ngrok`

Reply:
Good catch, I'd say not a nit, and we will want to change that.

Reply:
@deroine, fixed in https://github.com/ngrok/ngrok-ingress-controller/pull/72/commits/8fc64dfce1b49f192ed48dc4ffec983bfe5d6933